src/app/modules/main/wallet_section/following_addresses/controller.nim

```diff
@@ -0,0 +1,66 @@
+import io_interface, chronicles
+import app/core/eventemitter
+import app_service/service/following_address/service as following_address_service
+
+const SIGNAL_FOLLOWING_ADDRESSES_UPDATED* = following_address_service.SIGNAL_FOLLOWING_ADDRESSES_UPDATED
```

Comment:
You don't need to setup a `const`, you can use  `following_address_service.SIGNAL_FOLLOWING_ADDRESSES_UPDATED` directly below